File: easy/ContainsDuplicate.cpp
#include <iostream>
#include <unordered_set>
#include <vector>

using namespace std;

class Solution {
public:
  bool hasDuplicate(vector<int> &nums) {
    return unordered_set<int>(nums.begin(), nums.end()).size() < nums.size();
  }
};

int main() {}

File: js/easy/ContainsDuplicate.js
class Solution {
	/**
	* @param {number[]} nums
	* @returns {boolean}
	*/
	hasDuplicate(nums) {
		const set = new Set();
		for (const num of nums) {
			set.add(num);
		}
		return (set.size !== nums.length);
	}

	
}

